lib/metadata/bucketclient/backend.js

```diff
@@ -120,7 +120,12 @@ class BucketClientInterface {
 
     listMultipartUploads(bucketName, params, log, cb) {
         this.client.listObject(bucketName, log.getSerializedUids(), params,
-            (err, data) => cb(err, JSON.parse(data)));
+            (err, data) => {
+                if (err) {
+                    return cb(err);
+                }
+                return cb(err, JSON.parse(data));
+            });
         return null;
```

Comment:
I know this isn't part of this PR, but maybe we should just return `this.client.listObject`.